electrumsv/tests/test_wallet.py

```diff
@@ -608,6 +613,142 @@ def test_legacy_wallet_loading(mock_wallet_app_state, storage_info: WalletStorag
     if expected_network in { "testnet", "regtest" }:
         Net.set_to(SVMainnet)
 
+@unittest.mock.patch('electrumsv.wallet.app_state', new_callable=_create_mock_app_state)
+def test_password_update(mock_wallet_app_state) -> None:
+    """
+    This is a regression test for the bug described at:
+    https://github.com/electrumsv/electrumsv/pull/1070
```

Comment:
Do not rely on an external source for the reason for something. You can state it here.